.github/workflows/dist.yaml

```diff
@@ -0,0 +1,30 @@
+name: dist-check
+
+on: [push, pull_request]
+
+jobs:
+  dist:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Build dist
+      run: |
+        python setup.py sdist
+    - name: Twine check
+      run: |
+        pip install twine
+        last_dist=$(ls -t dist/openml-*.tar.gz | head -n 1)
+        twine check $last_dist
+    - name: Install dist
+      run: |
+        last_dist=$(ls -t dist/openml-*.tar.gz | head -n 1)
+        pip install $last_dist
+    - name: PEP 561 Compliance
```

Comment:
I'm not 100% sure if this test is sufficient. We only test if it's installable, but not whether the installed package actually works. Having said that, as we don't compile anything, it's maybe sufficient.

Reply:
As far as I can tell, unless we're changing something about the structure of the package (i.e. include more than just `openml`), I don't think results should differ. To keep concerns separate and prevent additional load on the server, I decided not to run an additional test suite here. I'd say we wait until we encounter a problem from this before we add it.

Reply:
Makes sense.